src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -91,6 +92,7 @@
               (activity-center/notifications-fetch-pending-contact-requests)
               (activity-center/update-seen-state)
               (activity-center/notifications-fetch-unread-count)
+              (wallet-2/add-chains-to-db)
```

Comment:
get network info on login

Reply:
I guess we should rename it to `fetch-chains` as it actually does a `status-go` call. WDYT?

Reply:
Definitely